data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	"Live in a ship?" Aunaris repeats the sentence, puzzled.`
+			`	"In an Arach ship, for that week I worked. Taught me some things, the captain did, often repurposed into houses, their ships are," he explains. "Their smallest ship, we could buy, save time and money otherwise spent building a home, it would."`
+			`	They talk for a bit in private, and after a few minutes Inturi apparently managed to fully explain his idea and convince her.`
+			`	They head back to the real estate agency, and arrange the purchase of the plot of land. Inturi then asks that you transport him to <destination>, where the basic Arach ship model is sold, and after buying it he will return here with the new "house."`
+			`	Aunaris will remain here to finish sorting out all the paperwork, and will be waiting by the property once you two return.`
+				accept
+
+
+
+mission "Saryd Couple 4"
+	name "Test Drive, Final Drive"
+	description "Escort Inturi's ship, which will be repurposed into a home for him and Aunaris, to <destination>."
+	landing
+	to offer
+		has "Saryd Couple 3: done"
+	source "Saros"
+	destination "Far Garden"
+	on offer
+		conversation
+			`Inturi heads right to the shipyard, and you help him with the purchase itself. He says that alhough he has worked in many ships over the years, he never went to the shipyard for more than picking up some parts for repair, or to polish a newly produced model.`
+			`	Only when you're with him inside the Arach vessel is that you think of asking if he even knows how to fly a ship, but he reassures you that that won't be a problem.`
+			`	"To work as a copilot for an old friend, my first job was. Worry about me crashing the ship, you need not," he jokes.`
+			`	He says he will wait for your lead, and follow you to <destination> once you depart.`
+				accept
+	npc accompany save
+		personality escort timid
+		government "Coalition"
+		ship "Arach Courier" "Boop"
+	on visit
+		dialog `You have reached <planet>, but Inturi's ship hasn't arrived yet. Better depart and wait for it to get here.`
+	on complete
+		"coalition jobs" ++
+		event "saryd couple follow-up" 51 96
+		conversation
+			`You help Inturi park the ship on the plot of land, but since the Arach Courier takes up almost the entire area, your own ship has to land on a regular landing pad.`
+			`	After taking a transport up the mountain, you see that some workers are already there, and are starting to remove ship components that won't be needed, like its engines and the hyperdrive. A house doesn't need to be able to traverse the void of space, after all.`
+			`	The couple comes meet up with you, "In a day or two, after some shopping we do, fully furnished the house will be," Aunaris says, looking at the ship.`
+			`	"Helped us a lot in getting it all sorted out you have, Captain <last>. Focus on working hard to pay you back in full, we will," Inturi adds.`
+			choice
+				`	"You don't need to worry about paying me, I helped because I wanted to, not for a payday."`
```

Comment:
```suggestion
				`	"You don't need to worry about paying me. I helped because I wanted to, not for a payday."`
```
You might be able to use Amazinite's new payment in conversation feature here. Seems like you have it all worked out here, but just thought I would let you know.

Reply:
when that is merged